spec/features/collections/collection_limited_ondemand_products_spec.rb

```diff
@@ -1,36 +1,31 @@
 require 'rails_helper'
 
-describe "Limited Collections" do
+describe 'Limited Collections' do
   extend Helpers::CollectionHelpers
 
-
-  before :all do
-    Capybara.reset_sessions!
-    load_page :search, q: 'C14758250-LPDAAC_ECS', close_banner: true
-  end
-
-  pending 'when the number of granules exceeds the collection limit (ASTER for example) in a project' do
+  context 'when the number of granules exceeds the collection limit (ASTER for example) in a project' do
     before :all do
-      target_collection_result('ASTER L1A Reconstructed Unprocessed Instrument Data V003').click_link 'Add collection to the current project'
+      load_page :search, project: 'C14758250-LPDAAC_ECS', authenticate: 'edsc'
       find('div[class="toolbar-secondary"]').click_link 'My Project'
-    end
+      wait_for_xhr
```

Comment:
`wait_for_xhr` is called as part of `load_page` if this is necessary there is something going on.